sounds/LICENSE

```diff
@@ -0,0 +1,7 @@
+message.wav, online.wav
+Copyright 2015, qso (qso@users.sf.net)
```

Comment:
Both sounds are created by myself. 
message.wav is just a short part of the delay feedback of a few drum hits through a filter, all done by myself.
online.wav are recorded leaves of a plant next to me.